DDM potential fraud alerts (#195)
department-of-veterans-affairs/developer-portal-content: content/direct-deposit-management/release-notes/2025-02-05.md

> Dependents are no longer allowed to make updates to direct deposit information. This change is implemented to prevent potential data conflicts with downstream systems.

Reviewer: ```suggestion
The Direct Deposit Management API now enforces stricter validation and fraud detection measures.
```

> The Direct Deposit API checks the ID of the requester to determine if they are a Veteran or dependent. If they are a dependent, or if Veteran/dependent status cannot be determined, the API generates a HTTP 422 error to notify the requester that they are not allowed to make updates.  

Reviewer: ```suggestion
- Dependents can no longer update direct deposit information to prevent data conflicts with downstream systems.
- The API checks the requester's ID to determine if they are a Veteran or dependent. If the requester is a dependent or their status is unclear, the API returns an HTTP 422 error notifying the requester that they aren't allowed to make updates.
- If a Veteran’s account is flagged for potential fraud by the VA source system, the API response includes a fraud alert in the header.
```

> 

Reviewer: ```suggestion
```

> When a Veteran submits updates to their direct deposit account, the API detects whether the account is flagged by the VA source system for prior possible fraud activity. A frauld alert entry is added to the header of the API response in this case.  

Reviewer: ```suggestion
```
